azure/Guidelines.md

```diff
@@ -725,19 +727,28 @@ You MAY support preflight requests...
 Preflight requests?  Often supported for CORS but could be used for any potentially expensive request.  Consider "EXPECT: 100-continue" for other requests that will return 100 Continue if the conditions are satisfactory or 417 Expectation Failed otherwise. Useful for conditionaly requests with large payloads. You should return the same error code that the service should use. 
 SHOULD provide documentation on what preflight checks will be validated.
 
-AWS uses this for S3 API
-
 
 -	Status codes
-o	GET: if the comparison fails, return 304 Not Modified (consider also returning Expires/Cache-Control/Age headers for caching scenarios)
-o	PUT/POST/DELETE: if the comparison fails, return 412 Precondition Not Met
-o	Consider forcing conditional headers on resource mutation, then use 428 Precondition Required if they're not present.
+* GET: if the comparison fails, return 304 Not Modified (consider also returning Expires/Cache-Control/Age headers for caching scenarios)
+* PUT/POST/DELETE: if the comparison fails, return 412 Precondition Not Met
+* Consider forcing conditional headers on resource mutation, then use 428 Precondition Required if they're not present.
 
 -	If-Match header - should support multiple values that are Or-ed together per HTTP/1.1.
 
 
+### Distributed Tracing & Telemetry
+Azure Core guidelines specify that you support telemetry data through both the ```User-Agent``` header, ```X-MS-UserAgent``` header, and Open Telemetry. 
```

Comment:
Most people in the company don't know what "Azure Core" refers to and would probably assume you meant "Core Azure" which is way different.
```suggestion
Azure SDK client guidelines specify that client libraries must send telemetry data through the ```User-Agent``` header, ```X-MS-UserAgent``` header, and Open Telemetry. 
```